ros_gazebo_gym_panda_example_params:
  task_env: "PandaReach-v1"
  # task_env: "PandaPickAndPlace-v1"
  # task_env: "PandaPush-v1"
  # task_env: "PandaSlide-v1"
  # ros_ws_abspath: "/home/user/simulation_ws"

  # Q-learning parameters
  alpha: 0.01 # Learning Rate
  gamma: 1.0 # Future rewards value 0 none 1 a lot
  n_episodes: 100
  n_steps: 1000

  # Inference parameters
  inference: False
  inference_n_episodes: 1
  inference_n_steps: 5
